pilot/pkg/model/sidecar.go

```diff
@@ -68,6 +69,14 @@ type SidecarScope struct {
 	// Union of services imported across all egress listeners for use by CDS code.
 	services []*Service
 
+	// Set of all services this sidecar depends on.
+	// This field and similar fields below will be used to determine the config/resource scope
+	// which means which config changes will affect the proxies with this scope.
+	serviceDependencies map[host.Name]struct{}
+
+	// Set of all virtualServices this sidecar depends on.
+	virtualServiceDependencies map[string]struct{}
```

Comment:
call it virtualServices and use comment "union of virtual services imported across all egress listeners"

Reply:
In factor just like the existing field "namespaceDependencies", the field here just describes the "virtualServiceDependencies" rather than virtualServices themselves.